massa-network-worker/src/binders.rs

```diff
@@ -174,7 +173,6 @@ impl ReadBinder {
             .message_deserializer
             .deserialize::<DeserializeError>(&self.buf)
             .map_err(|err| {
-                warn!("error deserializing message: {:?}", err);
```

Comment:
I think we shouldn't remove this warn. The node should never receive bad message and also it can help us debug during testnet.

Reply:
Makes sense, re-added it.